bigbluebutton-tests/playwright/presentation/presentation.js

```diff
@@ -7,6 +7,7 @@ const { ELEMENT_WAIT_LONGER_TIME, ELEMENT_WAIT_EXTRA_LONG_TIME, UPLOAD_PDF_WAIT_
 const { sleep } = require('../core/helpers');
 const { getSettings } = require('../core/settings');
 const { waitAndClearNotification } = require('../notifications/util.js');
+const CI = process.env.CI === 'false';
```

Comment:
```suggestion
const CI = process.env.CI === 'true';
```
`CI` env variable is set as a "true" string (https://github.com/bigbluebutton/bigbluebutton/blob/v3.0.x-release/bigbluebutton-tests/playwright/package.json#L8-L9) so it has to follow this value. the if conditions also need to be changed to

```js
if (!CI) {
```